### ISSUE TITLE
investigate, why it trains so fast?

### ISSUE THREAD

**Issue by Exception4U** [Exception4U/DQN-tensorflow#1]

**Comment by ghost** [comment]
Do you mean it gets much faster with  tf 0.12, gym>0.7 and python3 than the original devsisters configuration?

**Comment by Exception4U** [comment]
nope, episodes are rendered very fast during training .. I suspect clock rate is controlled in the non-rendering mode for the simulator.

**Comment by c433li** [comment]
Ok, thanks.
